### PR TITLE
[493] Pull image from private GitHub package

### PR DESCRIPTION
**Pull request by saliceti**
## What
It is required to close the Github package temporarily so we need to provide an authenticated user and token.

**Depends on: https://github.com/DFE-Digital/terraform-modules/pull/61**

## How to review
- Create PAT
- Add it to keyvault
- Deploy a **new** image

### CODE REVIEW

File: terraform/application/application.tf [hunk 1]
```diff
@@ -34,5 +34,7 @@ module "web_application" {
   kubernetes_config_map_name = module.application_configuration.kubernetes_config_map_name
   kubernetes_secret_name     = module.application_configuration.kubernetes_secret_name
 
-  docker_image = "${local.docker_repository}:${var.docker_image_tag}"
+  docker_image    = "${local.docker_repository}:${var.docker_image_tag}"
+  github_username = module.infrastructure_secrets.map.GITHUB-USERNAME
+  github_pat      = module.infrastructure_secrets.map.GITHUB-PAT
```
Comment (your) [inline terraform/application/application.tf:39]: This may need to change to `GITHUB-PERSONAL-ACCESS-TOKEN` now

**Review by your — COMMENTED** [review]
Looking good, waiting for a change and the dependency before approving

File: terraform/application/application.tf (hunk 1 above)
Comment (saliceti) [reply terraform/application/application.tf]: done!

**Review by your — APPROVED** [review]

**Comment by saliceti** [comment]
[Pull image from private Github package](https://trello.com/c/8iE503j7/493-pull-image-from-private-github-package)

**Pull request merged**